README.md

```diff
@@ -34,7 +34,7 @@ const winston = require('winston');
 const { Chitragupta } = require('chitragupta');
 
 const jsonLogFormatter = winston.format.printf( ({ level, message, ...meta}) => {
-  const options = {level: level, message: message, meta: meta};
```

Comment:
data will go in the meta block itself. Also, please highlight the changes in all the examples